api/main.go

```diff
@@ -38,9 +56,60 @@ func InitDb() *gorm.DB {
 		db.CreateTable(&Venues{})
 		db.Set("gorm:table_options", "ENGINE=InnoDB").CreateTable(&Venues{})
 	}
+
+	if os.Getenv("TEST") == "1" {
```

Comment:
This is filthy. We need to find a way to move this out of the production code and into the test environment.